README.md

```diff
@@ -305,4 +308,5 @@ are required. After that the configuration will be removed.
 [node]: https://nodejs.org/
 [ns]:   https://nodesource.com/
 [pivotal]: https://www.pivotalagency.com.au/
+[release secrets]: https://github.com/nodejs-private/secrets/tree/master/build/release/.gpg
```

Comment:
```suggestion
[release secret members]: https://github.com/nodejs-private/secrets/tree/master/build/release/.gpg
```
^--- maybe make that its a link to the membership list more explicit? Generally, we link to the gh team with access, so the list in the README.md can be audited against the actual membership, and the .gpg directory is the equivalent for this group.